src/components/Pickers/Lookup/Lookup.js

```diff
@@ -367,7 +376,12 @@ class Lookup extends React.PureComponent {
 			return;
 		}
 		const rect = e && e.target && e.target.parentElement.getBoundingClientRect();
-		if (!this.state.optionsCache[this.state.search] || !some(this.state.optionsCache[this.state.search].options)) {
+		const availableValues = _(_.values(this.state.optionsCache))
```

Comment:
`_(this.state.optionsCache).values()....`

Reply:
Fair enough, changed